src/algebra/ring.lean

```diff
@@ -298,6 +299,17 @@ infixr ` →+* `:25 := ring_hom
 instance {α : Type*} {β : Type*} [semiring α] [semiring β] : has_coe_to_fun (α →+* β) :=
 ⟨_, ring_hom.to_fun⟩
 
+instance {α : Type*} {β : Type*} [semiring α] [semiring β] : has_coe (α →+* β) (α →* β) :=
+⟨ring_hom.to_monoid_hom⟩
+
+instance {α : Type*} {β : Type*} [semiring α] [semiring β] : has_coe (α →+* β) (α →+ β) :=
+⟨ring_hom.to_add_monoid_hom⟩
+
+@[squash_cast] lemma coe_monoid_hom {α : Type*} {β : Type*} [semiring α] [semiring β] (f : α →+* β) (a : α) :
```

Comment:
Should these also be in the default simp set? (Or is that done by default? I don't remember.)